sig-scheduling/README.md

```diff
@@ -85,6 +85,9 @@ The following [subprojects][subproject-definition] are owned by sig-scheduling:
 - **Owners:**
   - [kubernetes-sigs/kube-scheduler-wasm-extension](https://github.com/kubernetes-sigs/kube-scheduler-wasm-extension/blob/main/OWNERS)
 ### kueue
+- **Leads:**
+  - Aldo Culquicondor (**[@alculquicondor](https://github.com/alculquicondor)**), Google
+  - Yuki Iwai (**[@tenzen-y](https://github.com/tenzen-y)**), CyberAgent, Inc.
```

Comment:
I added these lines based on this template: https://github.com/kubernetes/community/blob/c853773afa01f2429b045e4c46372721962d87ef/generator/sig_readme.tmpl#L88-L96

Reply:
isn't this auto-generated?

Reply:
That is correct, but it looks like it was all done correctly otherwise there would be a template validation error :)

@tenzen-y, next time if you make a change to sigs.yaml and then run `make` from the root of the repo, it will update it everywhere it needs to 👍

Reply:
Oh, I didn't know that! Thank you for letting me know!